txnbuild/path_payment.go

```diff
@@ -6,13 +6,15 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
-// PathPayment represents the Stellar path_payment_strict_receive operation. See
-// https://www.stellar.org/developers/guides/concepts/list-of-operations.html
-type PathPaymentStrictReceive PathPayment
+// PathPayment represents the Stellar path_payment operation. This operation was removed
+// in Stellar Protocol 12.
+// Deprecated: This operation is deprecated in favour of PathPaymentStrictReceive,
```

Comment:
I think it's worth saying it's renamed instead of deprecated when we're explaining why or how it has been deprecated. We already note that it is deprecated at the beginning of the comment.
```suggestion
// Deprecated: This operation was renamed to PathPaymentStrictReceive,
```